llama-index-core/llama_index/core/indices/struct_store/sql_query.py

```diff
@@ -359,10 +368,14 @@ def _query(self, query_bundle: QueryBundle) -> Response:
             partial_synthesis_prompt = self._response_synthesis_prompt.partial_format(
                 sql_query=sql_query_str,
             )
+            partial_refine_prompt = self._refine_synthesis_prompt.partial_format(
+                sql_query=sql_query_str,
```

Comment:
hmm, the `DEFAULT_REFINE_PROMPT` does not have a `sql_query` variable

Reply:
You're right, it's been fixed